apps/condo/domains/miniapp/constants.js

```diff
@@ -16,8 +16,10 @@ const APP_TYPES = [
 // GLOBAL B2B APPS FEATURES
 const GLOBAL_FEATURE_GRAPHQL_TYPE = 'B2BAppGlobalFeature'
 const MAP_GENERATION_FEATURE = 'PropertyMapGeneration'
+const ATTACH_RECORD_TO_TICKET_FEATURE = 'AttachRecordToTicket'
```

Comment:
Should we add `call` prefix to record here?

Reply:
Added